seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-elasticsearch7/src/main/java/org/apache/seatunnel/flink/elasticsearch/sink/ElasticsearchOutputFormat.java

```diff
@@ -64,27 +62,14 @@ public ElasticsearchOutputFormat(Config userConfig, ElasticsearchSinkFunction<T>
     @Override
     public void configure(Configuration configuration) {
         List<String> hosts = config.getStringList(HOSTS);
-        Settings.Builder settings = Settings.builder();
-
-        config.entrySet().forEach(entry -> {
```

Comment:
Seem like you remove custom setting for elasticsearch, please add it back.

Reply:
Elasticsearch will not support settings on rest client, neither 7.x RestHighLevelClient or 8.x ElasticsearchClient, for what all them based on RestClient doesn't support it.